libs/sdk-ui-theme-provider/src/ThemeProvider/ThemeProvider.tsx

```diff
@@ -27,6 +27,17 @@ export interface IThemeProviderProps {
      * component tree.
      */
     workspace?: string;
+
+    /**
+     * Theme object to use instead of one loaded from workspace
+     */
+    adhocTheme?: ITheme;
+
+    /**
+     * If true, global theme will be automatically cleared during component unmount.
+     * Default value is false
+     */
+    autoClear?: boolean;
```

Comment:
Should this be true by default? I mean why wouldn't you want this?

Reply:
removed and done by default